index.html

```diff
@@ -0,0 +1,35 @@
+<!DOCTYPE html>
+<html lang="en">
+    <head>
+        <meta charset="UTF-8">
+        <title>Personal Portfolio</title>
+        <meta name="viewport" content="width=device-width, initial-scale=1.0">
+        <link rel="stylesheet" href="styles.css">
+        <link rel="preconnect" href="https://fonts.googleapis.com">
+        <link rel="preconnect" href="https://fonts.gstatic.com" crossorigin>
+        <link href="https://fonts.googleapis.com/css2?family=Crete+Round&display=swap" rel="stylesheet">
+        <link rel="preconnect" href="https://fonts.googleapis.com">
+        <link rel="preconnect" href="https://fonts.gstatic.com" crossorigin>
+        <link href="https://fonts.googleapis.com/css2?family=Crete+Round&family=Poppins:wght@100&display=swap" rel="stylesheet">
+    </head>
+    <body>
+        <nav class="nav-bar">
+            <a class="logo" href="#">Snoplezik</a>
+            <a class="hamburger" href="#"><div></div><div></div><div></div></a>
+        </nav>
+        <header class="headline">
+            <img class="background" src="images/Illustration-header.png" alt="">
+            <h1>Hey There.<br>I'm Santiago</h1>
+            <h2>I am a Software Developer</h2>
+            <p>I can help you build a product , feature or website Look through some of my work and experience! If you like what you see and have a project you need coded, don’t hestiate to contact me.</p>
+            <ul class="social">
+                <li><a href=""><img src="images/GitHub/Tertiary/Icons/Frame.png" alt="GitHub"></a></li>
+                <li><a href=""><img src="images/Twitter/Tertiary/Icons/Frame.png" alt="Twitter"></a></li>
+                <li><a href=""><img src="images/Peace/Tertiary/Icons/angelist1.png" alt="Peace"></a></li>
+                <li><a href=""><img src="images/M/Tertiary/Icons/Vector.png" alt="M"></a></li>
+                <li><a href=""><img src="images/Linkedin/Tertiary/Icons/Frame.png" alt="Linkedin"></a></li>
+            </ul>
+        </header>
```

Comment:
- [ ] This section should be using the `section` tag instead of `header` kindly fix this so it has the right tag